blazor/pdfviewer/getting-started/wsl-application.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Getting Started with PDF Viewer in Blazor WSL mode | Syncfusion
+description: Learn how to getting started with PDF Viewer control in Blazor WSL (Windows Subsystem for Linux) mode. 
+platform: Blazor
+control: PDF Viewer
+documentation: ug
+---
+
+# Getting Started with Blazor PDF Viewer Component in WSL mode
+
+To run the Syncfusion Blazor PDF Viewer in WSL (Windows Subsystem for Linux) mode, you will need to take the following steps:
+
+**Step 1:** Enable the Windows Subsystem for Linux and the Virtual Machine Platform.
+
+![Create-new-blazor-wsl-app](GettingStarted_images/turn-features.png)
+
+To enable the Windows Subsystem for Linux (WSL) and the Virtual Machine Platform on Windows, follow these steps:
+
+Open the Start menu and search for `Control Panel`. Click on `Programs`, then click on `Turn Windows features on or off`. Scroll down and check the box next to `Windows Subsystem for Linux` and `Virtual Machine Platform`. Click `OK` and restart your machine.
+
+After your computer restarts, you will be able to install a Linux distribution from the Microsoft Store, such as Ubuntu, and run Linux commands directly in Windows.
+
+**Step 2:** Install the `Ubuntu`
+
+![Create-new-blazor-wsl-app](GettingStarted_images/ubuntu-install.png)
+
+Ubuntu can be installed on a Windows machine through the Microsoft Store. Here are the steps to do so:
+
+Open the Start menu and search for `Control Panel`. Click on `Programs`, then click on `Turn Windows features on or off`. Scroll down and check the boxes next to `Windows Subsystem for Linux` and `Virtual Machine Platform`. Click `OK` and restart your machine.
+
+Once the installation is complete, open the Windows search. In the terminal, type `Ubuntu` and press enter. This will launch the Ubuntu terminal inside of Windows.
+
+On Ubuntu, create a new user with a username and password. 
+
+![Create-new-blazor-wsl-app](GettingStarted_images/username-password.png)
+
+**Step 3:** Install the dotnet framework for running the WSL (Windows Subsystem for Linux) in the project by running the following code one by one. 
+
+```
+    wget https://packages.microsoft.com/config/ubuntu/22.10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+
+    sudo dpkg -i packages-microsoft-prod.deb
+
+    rm packages-microsoft-prod.deb
+
+    sudo apt-get update && \
+    sudo apt-get install -y dotnet-sdk-6.0
+
+```
+
+Check the comment dotnet --info and it should come like below
+
+![Create-new-blazor-wsl-app](GettingStarted_images/dotnet-info.png)
+
+If the above result is not shown, Please run the comments below or follow the steps in the link below to uninstall and reinstall dotnet. 

```

Comment:
If the above result is not shown, please run the comments below or follow the steps in the link to uninstall and reinstall dotnet.